README.md

```diff
@@ -1571,6 +1571,7 @@ Update Time, 5 active automations, webhooks.
 
 ## Analytics, Events and Statistics
 
+   * [Dwh.dev](https://dwh.dev) - Data Cloud Observability Solution (Snowflake). Free for personal use.
```

Comment:
No visible pricing or plan information, so cant review this I am afraid.

Reply:
Fixed
https://dwh.dev/#pricing